app.js

```diff
@@ -0,0 +1,45 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var mongo = require('mongodb');
```

Comment:
These dependencies that are listed up here all come from the node_modules folder which means that dependencies are installed within the project itself. This is populated using the dependencies from package.json when you run `npm install`. That way, you aren't committing all of a projects dependencies to Github. However, you'll notice that in the package.json file, mongodb isn't listed as a dependency. If I were to download this and try to run it by using
`npm install`
`npm start`
It would crash. Whenever you install a dependency, use
`npm install --save dependencyName`